tests/setup_machine.sh

```diff
@@ -19,7 +19,7 @@ usage() {
 
 		Example:
 		${script_name} setup /mnt
-		${script_name} setup /mnt/hda /mnt/hdb
+		${script_name} setup /mnt/had /mnt/hdb
```

Comment:
Using some kind of dictionary to find potential typos doesn't apply here where acronym is of type `hd{a,b...z}`
```suggestion
		${script_name} setup /mnt/hda /mnt/hdb
```

Reply:
This suggestion was addressed as @Antuan-AE suggested, so it is solved.